events/views.py

```diff
@@ -2367,7 +2365,9 @@ def organiser_invigilator_index(request, role, status):
 
     if role == 'organiser':
         try:
-            collectionSet = Organiser.objects.select_related().filter(academic=AcademicCenter.objects.filter(state=State.objects.filter(resourceperson__user_id=user, resourceperson__status=1)), status=status)
+            states =  user.resource_person.prefetch_related().filter(resourceperson__status = 1)
```

Comment:
Is prefetch_related really required here. If so are you sure you are using it correctly?